packs/data/age-of-ashes-bestiary.db/xotani-the-firebleeder.json

```diff
@@ -581,7 +581,7 @@
                     "value": 1
                 },
                 "description": {
-                    "value": "<p>Huge, [[/r {8d6+8}[bludgeoning]]]{8d6+8 bludgeoning} plus [[/r {12d6}[fire]]]{12d6 fire}, Rupture 35</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SwallowWhole]</p>"
+                    "value": "<p>Huge, [[/r (8d6+8)[bludgeoning]]] plus [[/r 12d6[fire]]], Rupture 35</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SwallowWhole]</p>"
```

Comment:
```suggestion
                    "value": "<p>Huge, [[/r (8d6+8)[bludgeoning],12d6[fire]]], Rupture 35</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SwallowWhole]</p>"
```

Reply:
These would need to be curly-brace-enclosed